odfuzz/config.py

```diff
@@ -54,6 +55,11 @@ def urls_per_property(self):
     def ignore_restriction(self):
         return self._ignore_restriction
 
+    
+    @property
+    def method(self):
```

Comment:
please rename to something like http_method(s). Not sure if this property should have only one or possibly multiple values (get, delete... how then?

Reply:
ok